app/packages/klogs/src/components/Logs.tsx

```diff
@@ -340,11 +340,11 @@ const DocumentDetailsTable: FunctionComponent<{
                         </IconButton>
                       </Tooltip>
 
-                      <Tooltip title={`add filter: _exists_ ${key}`}>
+                      <Tooltip title={`add filter: _and_ _exists_ ${key}`}>
```

Comment:
To be consistent with the other filers, i'd not change the tooltip.

Reply:
Makes sense